packages/sdk/src/utils/chain-constants.ts

```diff
@@ -77,8 +80,8 @@ const getL1ContractsByNetworkName = (network: string): OEL1ContractsLike => {
     StateCommitmentChain: getDeployedAddress('StateCommitmentChain'),
     CanonicalTransactionChain: getDeployedAddress('CanonicalTransactionChain'),
     BondManager: getDeployedAddress('BondManager'),
-    OptimismPortal: '0x5b47E1A08Ea6d985D6649300584e6722Ec4B1383' as const,
-    L2OutputOracle: '0xE6Dfba0953616Bacab0c9A8ecb3a9BBa77FC15c0' as const,
+    OptimismPortal: getContractDefinition('OptimismPortal').address,
+    L2OutputOracle: getContractDefinition('L2OutputOracle').address,
```

Comment:
AFAICT `getContractDefinition` only reads the HH artifacts, which don't have an address. 

We might need to port `getDeployedAddress` to the bedrock package? Or you could use [this](https://github.com/ethereum-optimism/optimism/blob/fc2f5887c3e3a14a9c7b2ac11bd0f5bf34895c37/packages/contracts-bedrock/src/deploy-utils.ts#L294), if you're OK with importing the HRE?

Reply:
Absolutely not!!! Do not ever port getDeployedContractDefinition to bedrock.  We need to delete the old one.   It causes major issues with code splitting.

But yea you are right I just replaced it with getting addy directly